src/nls/it/strings.js

```diff
@@ -451,8 +464,11 @@ define({
     "BASEURL_ERROR_HASH_DISALLOWED"     : "L’URL di base non può contenere hash come \"{0}\".",
     "BASEURL_ERROR_INVALID_CHAR"        : "I caratteri speciali come '{0}' devono essere %-encodate.",
     "BASEURL_ERROR_UNKNOWN_ERROR"       : "Trovato un carattere sconosciuto durante l'analisi del URL di base",
+    
+    // Strings for Pane.js
     "EMPTY_VIEW_HEADER"                 : "<em>Aprire un file mentre questo riquadro è attivo</em>",
-
+    "FLIPVIEW_BTN_TOOLTIP"              : "Capovolgere questa vista per {0 } riquadro",
```

Comment:
`Spostare questa vista nel riquadro {0}` (`{0}` gets replaced with the strings at [lines 240-243](https://github.com/Denisov21/brackets/blob/patch-40/src/nls/it/strings.js#L240-L243))